Mastodon.API.Tests/LinkTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using NUnit.Framework;
+using System.Net.Http.Headers;
+using System.Net;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Reflection;
+
+namespace Mastodon.API.Tests
+{
+    [TestFixture]
+    public class LinkTest
+    {
+        [Test]
+        public void createNextAndPrevLinkTest()
+        {
+            var str = TestUtils.GetResource("Mastodon.API.Tests.Resources.next_and_prev_link");
+            var link = Link.createLinkFromHeaderLinkValue(str);
+            Assert.NotNull(link);
+            Assert.AreEqual(1860868, link?.MaxId);
+            Assert.AreEqual(1874433, link?.SinceId);
+        }
+
+        [Test]
+        public void createNextLinkTest()
+        {
+            var str = TestUtils.GetResource("Mastodon.API.Tests.Resources.next_link");
+            var link = Link.createLinkFromHeaderLinkValue(str);
+            Assert.NotNull(link);
+            Assert.AreEqual(1860868, link?.MaxId);
```

Comment:
Since we Assert.NotNull in the previous line, if we have made it to this line we already know that link is not null so we probably wouldn't need to use the ? operator at this point.